scripts/devnet/docker-compose.yml

```diff
@@ -13,7 +13,7 @@ services:
   lotus_init:
     image: ${LOTUS_IMAGE}
     volumes:
-      - filecoin-proofs:${FILECOIN_PROOFS_DIR}
```

Comment:
I'd rather we keep the host clean, without any intermediary artefacts generated by running the devnet. We could achieve this with, e.g., populating the volume from a container or putting the filecoin proofs in the Docker images used within the devnet. 

What do you think?

Reply:
@LesnyRumcajs Changed to keep the `filecoin-proofs` volume and manipulate CI cache with extra steps